animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -314,3 +340,38 @@ def calculateCrossProducts(Vector3DList a, Vector3DList b):
 
     return result
 
+def findNeighboursEdges(Vector3DList vertices, EdgeIndicesList edges):
```

Comment:
Pass the length of `vertices` directly.

Reply:
Okay. The function's name `findNeighboursEdges` or `findNeighbours`?

Reply:
For consistency, make it `getLinkedVertices`.

Reply:
Renamed.